stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/MessagesScreen.kt

```diff
@@ -133,16 +132,6 @@ public fun MessagesScreen(
     val attachmentsPickerViewModel =
         viewModel(AttachmentsPickerViewModel::class.java, factory = factory)
 
-    val currentState = listViewModel.currentMessagesState
```

Comment:
These state reads were way to high, causing the whole screen to recompose when pretty much anything changed.